array_apply.json

```diff
@@ -63,7 +63,13 @@
                         "optional": true,
                         "default": null
                     }
-                ]
+                ],
+                "returns": {
+                    "description": "Return the value that should be stored in the data cube.",
```

Comment:
> data cube

array_apply works on and returns array

Reply:
Good catch, will be fixed in the next commit.